### PR TITLE
Issue #271, audit plugin

### PR DESCRIPTION
**Pull request by kirjaamo**
First draft of how to use the AUDIT plugin.
Can you check for content, I'll check for layout.

### CODE REVIEW

File: galeracluster/source/documentation/audit-log-plugin.rst [hunk 1]
```diff
@@ -0,0 +1,821 @@
...
+=================
+ Audit Log Plugin
+=================
+
+.. index::
+   pair: Descriptions; Audit Log Plugin
+.. index::
+   pair: Logs; Audit Log
+
+Galera Audit Log Plugin monitors and logs connection and query activity performed on a specific server. Information on the activity is stored in the XML log file, where each event will have a ``NAME`` field, a unique ``RECORD_ID`` field and a ``TIMESTAMP`` field. This implementation is alternative to the MySQL Enterprise Audit Log Plugin.
```
Comment (byte) [inline galeracluster/source/documentation/audit-log-plugin.rst:65]: The Galera Audit Log Plugin...

Comment (byte) [reply galeracluster/source/documentation/audit-log-plugin.rst:65]: I'm not even sure if we should call it the Galera Audit Log Plugin. I think "The Audit Log Plugin" will suffice

File: galeracluster/source/documentation/audit-log-plugin.rst [hunk 2]
```diff
@@ -0,0 +1,821 @@
...
+This variable is used to specify the ``ident`` value for syslog. This variable has the same meaning as the corresponding parameter described in the `syslog(3) manual <https://linux.die.net/man/3/syslog>`_.
+
+.. csv-table::
+   :class: doc-options
+
+   "Command-line Format", "``--audit_log_syslog_ident``"
+   "Variable Scope", "Global"
+   "Dynamic Variable", "No"
+   "Variable Type", "String"
+   "Default Value", "``percona-audit``"
```
Comment (byte) [inline galeracluster/source/documentation/audit-log-plugin.rst:751]: percona-audit?

**Review by byte — CHANGES_REQUESTED** [review]
Overall, I think we call it: The Audit Log Plugin

I also think it is missing attribution to Percona. I am not entirely sure we can take copyright of something that comes from Percona (please include the license)

Next up, I have to ask, did you test this with Galera Cluster or should I test every option to make sure?

Thanks

File: galeracluster/source/documentation/audit-log-plugin.rst (hunk 2 above)
Comment (kirjaamo) [reply galeracluster/source/documentation/audit-log-plugin.rst:751]: What is the default value?

File: galeracluster/source/documentation/audit-log-plugin.rst (hunk 1 above)
Comment (kirjaamo) [reply galeracluster/source/documentation/audit-log-plugin.rst]: Fixed and pushed.

**Comment by kirjaamo** [comment]
I did not test any options with Galera Cluster. I don't have it installed and I think I dpn't have the expertise (or budget) or to do so.

**Comment by kirjaamo** [comment]
Which license do you want me to add? Or is it enough to say that the topic is adapted from Percona documentation?

**Comment by byte** [comment]
@kirjaamo so... I talked to Sakari, and license for anything you take from Percona should not be "Copyright Codership"

You will notice they use the Creative Commons Attribution-ShareAlike 2.0 license

ref: https://github.com/percona/percona-server/blob/8.0/doc/source/copyright.rst

and also https://creativecommons.org/licenses/by-sa/2.0/

So, please do the needful in terms of license change

**Comment by byte** [comment]
> I did not test any options with Galera Cluster. I don't have it installed and I think I dpn't have the expertise (or budget) or to do so.

as for this; I need to test this first before I merge; so once you've made the license change, you could reassign this to me, and I'll handle the testing before merging and pushing

**Review by byte — COMMENTED** [review]
OK, this is good. Now, I haven't had the time to try this out yet

**Comment by temeo** [comment]
test_pr

**Pull request merged**